xalia/app.manifest

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>

```

Comment:
Probably makes no difference what the `name` attribute is here since this is mainly boilerplate, but might be nice to have it as `xalia` or something.

Reply:
Yeah I didn't fully read the template, thanks for catching that.